airbyte-cdk/python/airbyte_cdk/sources/file_based/file_based_stream.py

```diff
@@ -194,7 +202,7 @@ async def _infer_file_schema(self, file: RemoteFile) -> Mapping[str, Any]:
             )
         except Exception as exc:
             raise SchemaInferenceError(
-                f"Error inferring schema for file: {file.uri}"
+                f"Error inferring schema for file: {file.uri}. Is the file valid {self.config.file_type}?"
```

Comment:
Can schema inference exception be caused by the file type? In which case? If it's not always the case, should we refine this error message?